scalableminds/webknossos
# Fix zarr streaming datasource-properties.json generation for non-wkw/zarr datasets

Got a match error here for a neuroglancer precomputed dataset. Since the values don’t differ, it should be fair to just match against the generic SegmentationLayer and DataLayer, not individual input data types.

### Steps to test:
- Import a neuroglancer precomputed dataset
- visit its corresponding zarr streaming properties route: `http://localhost:9000/data/zarr/sample_organization/<dataset-name>/datasource-properties.json?token=secretSampleUserToken`
- Should get a json, not a match error exception

------
